src/measure_theory/borel_space.lean

```diff
@@ -1167,6 +1291,67 @@ lemma measurable_of_tendsto_metric {f : ℕ → α → β} {g : α → β}
   measurable g :=
 measurable_of_tendsto_metric' at_top hf lim at_top_countable_basis (λ i, countable_encodable _)
 
+lemma ae_measurable_of_tendsto_metric_ae {μ : measure α} {f : ℕ → α → β} {g : α → β}
+  (hf : ∀ n, ae_measurable (f n) μ)
+  (h_ae_tendsto : ∀ᵐ x ∂μ, filter.at_top.tendsto (λ n, f n x) (nhds (g x))) :
+  ae_measurable g μ :=
+begin
+  by_cases hβ : nonempty β,
+  { let p : α → (ℕ → β) → Prop := λ x f', filter.at_top.tendsto (λ n, f' n) (nhds (g x)),
```

Comment:
In general, we use `𝓝` instead of `nhds` (this needs `open_locale topological_space`). could you change it here and below?